modal/runner/shared/download.py

```diff
@@ -11,13 +13,16 @@
     models_volume,
 )
 
+logger = get_logger(__name__)
```

Comment:
Does this work at the top-level scope? Does the stub's env injection take effect here?

Reply:
Commented here on it: https://github.com/OpenRouterTeam/openrouter-runner/pull/48#discussion_r1451613369